hnn_core/gui/_viz_manager.py

```diff
@@ -489,32 +489,29 @@ def _get_ax_control(widgets, data, fig_idx, fig, ax):
     analysis_style = {'description_width': '200px'}
     layout = Layout(width="98%")
     simulation_names = tuple(data['simulations'].keys())
-    sim_name_default = simulation_names[-1]
-
-    if len(simulation_names) == 0:
-        simulation_names = [
-            "None",
-        ]
+    sim_index = 0
+    if not simulation_names:
+        simulation_names = ("None",)
+    else:
+        # Find the last simulation with a non-None 'net'
+        sim_index = next(
+            (idx for idx, sim_name in enumerate(simulation_names)
+             if data['simulations'][sim_name]['net'] is not None),
+            0  # Default value if no such simulation is found
+        )
```

Comment:
```suggestion
        # Find the last simulation with a non-None 'net'
        sim_index = next(
            (idx for idx, sim_name in 
             reversed(list(enumerate(simulation_names)))
             if data['simulations'][sim_name]['net'] is not None),
            0  # Default value if no such simulation is found
        )
```
This will find the last simulation instead of the first.